src/components/Divider/Divider.tsx

```diff
@@ -8,7 +8,11 @@ import { childrenExist, createShorthandFactory, customPropTypes, UIComponent } f
 
 /**
  * @accessibility
- * This is shown at the top.
+ * If there is no text provided inside divider, then "role='separator'" should be set.
+ *
+ *
+ * Other considerations:
+ *  - if separator contains text, then "role='separator'" cannnot be used because text is not narrated by VoiceOver (even with aria-label)
```

Comment:
This might be confusing - it tells the user what happens, but not how to fix.  Perhaps 'if the separator does contain text, then the aria-label / aria-labelledby should indicate it's a separator as well as the associated text'